cpp/dolfinx/fem/utils.cpp

```diff
@@ -147,15 +165,9 @@ fem::compute_integration_domains(fem::IntegralType integral_type,
     entities = entities.first(std::distance(entities.begin(), it1));
   }
 
-  std::vector<std::int32_t> entity_data;
-  switch (integral_type)
-  {
-  case IntegralType::cell:
-  {
-    entity_data.insert(entity_data.begin(), entities.begin(), entities.end());
-    break;
-  }
-  default:
+  auto cell_connectivity = [tdim, &topology]()
```

Comment:
Could have a more descriptive name, as it is only used for `facet` integrals, i.e get_cell_facet_connectivity?